opentracing-api/src/main/java/io/opentracing/ScopeManager.java

```diff
@@ -48,7 +39,7 @@
      *
      * <p>
      * If there is an {@link Scope non-null scope}, its wrapped {@link Span} becomes an implicit parent of any
-     * newly-created {@link Span} at {@link Tracer.SpanBuilder#startActive()} time (rather than at
```

Comment:
live above: `s/becomes an implicit parent/becomes an implicit CHILD_OF parent` (or something like that).